src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -408,6 +408,11 @@ const ProjectSelector = createReactClass({
   },
 });
 
+const StyledPlatformList = styled(PlatformList)`
```

Comment:
could we achieve this same effect minus all the random pixel values by aligning or floating the container right?